t/server/controller/contributor.t

```diff
@@ -0,0 +1,62 @@
+use strict;
+use warnings;
+
+use Cpanel::JSON::XS qw( decode_json );
+use MetaCPAN::Server::Test;
+use MetaCPAN::TestServer;
+use Test::More;
+
+my $server = MetaCPAN::TestServer->new;
+$server->index_packages;
```

Comment:
`index_packages` is already called in `t/00_setup.t:`  Do we need to do it again here?